.github/workflows/ci.yml

```
@@ -2,34 +2,36 @@ name: CI
 
 on:
   push:
-    branches: [master]
+    branches: [master, main]
   pull_request:
-    branches: [master]
+    branches: [master, main]
```

We don't have a `main` branch
```suggestion
    branches: [master]
```